sdk/fluidrelay/Azure.ResourceManager.FluidRelay/Azure.ResourceManager.FluidRelay.sln

```diff
@@ -3,17 +3,13 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.30309.148
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{85FA8590-3324-4532-BE25-88460DFFDFB8}") = "Azure.ResourceManager.FluidRelay", "src\Azure.ResourceManager.FluidRelay.csproj", "{CE7EE737-729B-4A01-9F87-A193D8846026}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.FluidRelay", "src\Azure.ResourceManager.FluidRelay.csproj", "{CE7EE737-729B-4A01-9F87-A193D8846026}"
```

Comment:
Please revert the change of this file, we don't need to add the test framework project to our sln.